src/Microsoft.PowerApps.TestEngine/TestInfra/PlaywrightTestInfraFunctions.cs

```diff
@@ -75,8 +75,14 @@ public async Task SetupAsync()
                 Timeout = testSettings.Timeout
             };
 
+            var browser = PlaywrightObject[browserConfig.Browser];
+            if (browser == null)
+            {
+                _singleTestInstanceState.GetLogger().LogError("Browser not supported by Playwright, for more details check https://playwright.dev/dotnet/docs/browsers");
+                throw new InvalidOperationException();
```

Comment:
Might be good to either throw a specific exception (create a new "BrowserNotSupportedException") or add a message to the exception